Std/Data/Array/Lemmas.lean

```diff
@@ -51,6 +51,10 @@ theorem not_mem_nil (a : α) : ¬ a ∈ #[] := nofun
 
 /-- # get lemmas -/
 
+theorem ext_get {a b : Array α} (h1 : a.size = b.size)
+    (h2 : ∀ i : Fin a.size, a[i] = b[i]) : a = b :=
+  Array.ext a b h1 (fun i h _ => h2 ⟨i, h⟩)
+
```

Comment:
Can you just drop this and use `apply Array.ext` directly below?

Reply:
bummer, i kind of liked how much simpler this one was stated. But I get that it appears there is no huge practical benefit